evil-pdf.el

```diff
@@ -106,7 +106,7 @@
     (kbd "<down-mouse-1>")  'pdf-view-mouse-set-region
 
     (kbd "C-c C-c") 'docview-mode
-    (kbd "C-c <tab>") 'pdf-view-extract-region-image
+    [?\C-c tab] 'pdf-view-extract-region-image ; workaround package-lint bug
```

Comment:
I don't think we should go down the route of twisting the code to make package-lint happy. package-lint is a helper, it has false positives and it's fine: we should not make the code uglier just for the sake of it.

Reply:
This *crashed* package-lint, and it would not continue it's analysis without the fix. This is not about silencing false positives.

Reply:
OK, but it is still just a helper. If the helper has an issue, it should not change our code which is fine.

Is the issue reported upstream?

Reply:
Fine, just revert the line and wait for it to be fixed.

Reply:
Let's not take lines in the sand. :)

We can both keep the code and report it upstream at the same time.

Packagelint is a tool but a very useful one. It's good to play nice with it even if it means tweaking the code just a little.

Think // ignore-linter type lines in other languages.

Reply:
Sure, we can keep the line and revert once it's fixed.  
Found the issue: https://github.com/purcell/package-lint/issues/96
@fbergroth : thanks for reporting.